### PR TITLE
Brush up several Equipment effects

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/equipment-effects/fists-of-the-ruby-phoenix/effect-apricot-of-bestial-might.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
     "_id": "WQ0DxUzMgAvo0zy9",
+    "folder": "xT95plXUpoOOApLs",
     "img": "icons/consumables/fruit/apricot-ripe-orange.webp",
     "name": "Effect: Apricot of Bestial Might",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p><span style=\"font-family:Roboto, sans-serif\">You gain 8 resistance to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. </span>You become @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}. If you perform an action with the auditory or linguistic traits, you must succeed at a @Check[flat|dc:5] or the action is lost.</p>"
```
Comment (TikaelSol) [inline packs/equipment-effects/fists-of-the-ruby-phoenix/effect-apricot-of-bestial-might.json:8]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain resistance 8 to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. You become @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}. If you perform an action with the auditory or linguistic traits, you must succeed at a @Check[flat|dc:5] or the action is lost.</p>"
```
Gotta check for these spans from copy/paste.

Control-shift-v will paste without formatting cruft like that (or copy HTML).

Also the effect doesn't do most of those things, it only adds the strike.

File: packs/equipment-effects/_folders.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        "_id": "rrVsnYxzh5wQeOwW",
+        "color": null,
+        "description": "",
+        "flags": {},
+        "folder": null,
+        "name": "Blood Lords",
+        "sort": 0,
+        "sorting": "a",
+        "type": "Item"
+    },
```
[the remark this answers is not in the document]
Comment (ammalagonc) [reply packs/equipment-effects/_folders.json:12]: All good -
I'll yeet the folders from this one.

[the remark this answers is not in the document]
Comment (ammalagonc) [reply packs/equipment-effects/_folders.json]: Alrighty - folders rare in the trash can

File: packs/equipment-effects/effect-antidote-greater.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Antidote (Greater)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Greater)]</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Fortitude saves against poisons</li>\n</ul>"
```
Comment (simonward) [inline packs/equipment-effects/effect-antidote-greater.json:7]: Consider updating the predicates for antidotes to `"item:trait:poison"`

File: packs/equipment-effects/effect-antidote-major.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Antidote (Major)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Fortitude saves against poisons</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>Immediately attempt a saving throw against one poison of 14th level or lower affecting you. If you succeed, the poison is neutralised. </li>\n</ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>You gain a +4 item bonus to Fortitude saving throws against poisons for 6 hours, and when you drink the antidote, you can immediately attempt a saving throw against one poison of 14th level or lower affecting you. If you succeed, the poison is neutralized.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-antidote-major.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>You gain a +4 item bonus to Fortitude saves against poisons.</p>"
```

File: packs/equipment-effects/effect-antiplague-greater.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Antiplague (Greater)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Greater)]</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Fortitude saves against diseases</li>\n</ul>"
```
Comment (simonward) [inline packs/equipment-effects/effect-antiplague-greater.json:7]: Consider updating the predicates to `"item:trait:disease"`

File: packs/equipment-effects/effect-antiplague-major.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Antiplague (Major)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Major)]</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Fortitude saves against diseases</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>Immediately attempt a saving throw against one disease of 14th level or lower affecting you. If you succeed, you are cured of the disease.</li>\n</ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Major)]</p>\n<p>You gain a +4 item bonus to Fortitude saves against diseases. When you drink the antiplague, you can immediately attempt a saving throw against one disease of 14th level or lower affecting you. If you succeed, you are cured of the disease.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-antiplague-major.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antiplague (Major)]</p>\n<p>You gain a +4 item bonus to Fortitude saves against diseases.</p>"
```

File: packs/equipment-effects/effect-apricot-of-bestial-might.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Apricot of Bestial Might",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-apricot-of-bestial-might.json:7]: The trait should be `deadly-d12`. In addition, the resistance and clumsy could be automated.

File: packs/equipment-effects/effect-architects-pattern-book.json [hunk 8]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Architect's Pattern Book",
     "system": {
         "description": {
-            "value": "<p>If your next action is to cast a @UUID[Compendium.pf2e.spells-srd.Item.Cozy Cabin], @UUID[Compendium.pf2e.spells-srd.Item.Planar Palace], or @UUID[Compendium.pf2e.spells-srd.Item.Resplendent Mansion] spell, you add a room to the structure that is up to 10 feet per side per rank of the spell. This room is outfitted with all the accoutrements for a particular type of recreation, determined by you when you cast the spell. Any character who spends at least 1 hour using this recreational facility and then sleeps a full 8 hours inside the location created by the spell is exceptionally well-rested. They regain double the amount of Hit Points they would normally receive for an 8-hour rest, and when they make the next day's preparations, they gain a +1 circumstance bonus to Athletics checks and Will saves for the next 12 hours.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Architect's Pattern Book]</p>\n<p>You gain a +1 circumstance bonus to Athletics checks and Will saves for the next 12 hours.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-architects-pattern-book.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Architect's Pattern Book]</p>\n<p>You gain a +1 circumstance bonus to Athletics checks and Will saves.</p>"
```

File: packs/equipment-effects/effect-ascendant-dragon-spirit.json [hunk 9]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Ascendant Dragon Spirit",
     "system": {
         "description": {
-            "value": "<p>Your unarmed Strikes deal an additional 1d6 poison damage and an additional 1d6 void damage.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Ascendant Dragon Spirit]</p>\n<p>Your unarmed Strikes deal an additional 1d6 poison damage and an additional 1d6 void damage.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-ascendant-dragon-spirit.json:7]: The DamageDice REs erroneously have `"category": "persistent"`.

File: packs/equipment-effects/effect-apricot-of-bestial-might.json [hunk 10]
```diff
@@ -36,8 +36,18 @@
                 "range": null,
                 "traits": [
                     "unarmed",
-                    "deadly-d10"
+                    "deadly-d12"
                 ]
+            },
+            {
+                "allowDuplicate": false,
```
Comment (simonward) [inline packs/equipment-effects/effect-apricot-of-bestial-might.json:43]:
```suggestion
                "inMemoryOnly": true,
```

File: packs/equipment-effects/effect-apricot-of-bestial-might.json [hunk 11]
```diff
@@ -4,13 +4,13 @@
     "name": "Effect: Apricot of Bestial Might",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain 8 resistance to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. You become @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-apricot-of-bestial-might.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain 8 resistance to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. You become clumsy 1.</p>"
```

File: packs/equipment-effects/effect-antidote-major.json [hunk 12]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Antidote (Major)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>Implemented effects:</p>\n<ul>\n<li>+4 item bonus to Fortitude saves against poisons</li>\n</ul>\n<p>Unimplemented effects:</p>\n<ul>\n<li>Immediately attempt a saving throw against one poison of 14th level or lower affecting you. If you succeed, the poison is neutralised. </li>\n</ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Antidote (Major)]</p>\n<p>You gain a +4 item bonus to Fortitude saves against poisons.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-antidote-major.json:7]: Predicate should be updated to `"item:trait:poison"`

**Pull request merged**